sources/packages/api/src/utilities/common.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * commonly used functions
+ */
+import * as dayjs from "dayjs";
+import * as utc from "dayjs/plugin/utc"; // import plugin
+import * as localizedFormat from "dayjs/plugin/localizedFormat"; // import plugin
+dayjs.extend(utc);
+dayjs.extend(localizedFormat);
+/**
+ * get utc date time now
+ * @returns date now in utc
+ */
+export const getUTCNow = () => {
+  return new Date(dayjs().utc().format());
```

Comment:
I noticed that dayjs has also a method like below. Is it the same as "new Date(dayjs().utc().format())"?
```
dayjs().utc().toDate()
```

Reply:
ya @andrewsignori-aot, both are the same, toDate() will be date object fed to new Date(), .format() is dates string fed to new Date(). new Date() accepts both date string and date object. both are correct